CHANGELOG.md

```diff
@@ -49,6 +49,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 * `jj op log` gained an option to include operation diffs.
 
+* A new global flag `--no-commit-transaction` lets you run a command without
```

Comment:
(Non-blocking) I'm thinking of alternative names.

`--no-publish-operation` might be a little better (?)

`--transient-operation`? In the same vein, `--disposable-operation` or `--provisional-operation` (but I prefer transient). `--ephemeral-operation` (thanks, thesaurus). 

`--dry-run` would be appropriate for many operations, but not for `fetch`, `push`, `git export`, and such.


-------------

We can absolutely decide or change our mind later, but I like "transient" and "ephemeral", I think.